### PR TITLE
Updated readme.md, Created contributing.md

### LINKED ISSUE #444: Separate contributing.md file in .github
File with how to contribute guidelines, needs to removed from readme.md and add link to the file in readme.md

### PR DESCRIPTION
**Pull request by ruchikabgosain**
Fixes #444
CC: @championshuttler

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,17 +6,7 @@ Welcome to Extension Workshop, a launchpad for building Firefox extensions! 🚀
 
 ## Updating Content
```
Comment (championshuttler) [inline README.md]: I think we can change it to #Contributing. WDYT @caitmuenster

File: README.md [hunk 2]
```diff
@@ -6,17 +6,7 @@ Welcome to Extension Workshop, a launchpad for building Firefox extensions! 🚀
...
-    - Make a fork of this repository.
-    - Clone your fork locally.
-    - Create a new branch to contain your change. Give your branch a descriptive name, such as update-distribution-options, or add-object-fit-example.
-    - Make your change. Please be sure to add your Github username to the lists for 'contributors' and 'last updated by', as well as the date of your update, for each file you change.
-    - Commit your change.
-    - Push the branch to your remote fork.
-    - Make a pull request to the original repo.
-
-If you are not sure how to complete the above steps, GitHub's [Fork a Repo guide](https://help.github.com/en/articles/fork-a-repo#fork-an-example-repository) is a good place to start.
+If you would like to update content or other resources on Extension Workshop, please refer to [Contributing.md](.github/contributing.md)
```
Comment (championshuttler) [inline README.md]:
```suggestion
If you would like to update content or other resources on Extension Workshop, please refer to [contributing.md](.github/contributing.md)
```

**Comment by ruchikabgosain** [comment]
did that! @championshuttler let me know if this works? 🍔

File: README.md (hunk 1 above)
Comment (caitmuenster) [reply README.md]: I think this is ok as is. :)

**Pull request merged**